src/IssueViz/Commands/IssueVizPackage.vsct

```diff
@@ -15,10 +16,11 @@
     <Buttons>
 
       <Button guid="guidIssueVizPackageCmdSet" id="IssueVisualizationToolWindowCommandId" priority="0x0100" type="Button">
-        <Parent guid="guidSHLMainMenu" id="IDG_VS_WNDO_OTRWNDWS1" />
-        <Icon guid="guidImages" id="sonarlint" />
+        <Parent guid="SonarLintCmdSet" id="SonarLintGroup"/>
+        <Icon guid="ImageCatalogGuid" id="TreeView" /> <!-- known VS icon ID -->
```

Comment:
The original command used the sonarlint icon cause it was under the Other Windows menu. Now there is no need for it so I opted for a default VS icon that looks a bit like the UI in that screen.

Btw, a very useful VS extension that allows you to see all VS icons: https://www.vsixgallery.com/extension/c38f3512-4653-4d97-a4c5-860a425209f6